### ISSUE TITLE
Webpack doesn't like @loadable/server 5.16.4

### ISSUE THREAD

**Issue by Judahmeek** [gregberge/loadable-components#1005]
```
webpack  | ERROR in ./node_modules/@loadable/server/dist/esm/loadable-server.esm.mjs 6:0-31
webpack  | Module not found: Error: Can't resolve 'lodash/uniq' in '/Users/judahmeek/apps/datacenters/node_modules/@loadable/server/dist/esm'
webpack  | Did you mean 'uniq.js'?
webpack  | BREAKING CHANGE: The request 'lodash/uniq' failed to resolve only because it was resolved as fully specified
webpack  | (probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
webpack  | The extension in the request is mandatory for it to be fully specified.
webpack  | Add the extension to the request.
```

### Results of `npx envinfo --system --binaries --npmPackages webpack,@loadable/component,@loadable/server,@loadable/webpack-plugin,@loadable/babel-plugin --markdown --clipboard`:

```bash
## System:
 - OS: macOS 14.4.1
 - CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
 - Memory: 2.19 GB / 16.00 GB
 - Shell: 5.9 - /bin/zsh
## Binaries:
 - Node: 18.16.0 - ~/.nvm/versions/node/v18.16.0/bin/node
 - Yarn: 4.1.1 - ~/.nvm/versions/node/v18.16.0/bin/yarn
 - npm: 9.5.1 - ~/.nvm/versions/node/v18.16.0/bin/npm
## npmPackages:
 - @loadable/babel-plugin: ^5.16.1 => 5.16.1 
 - @loadable/component: ^5.16.4 => 5.16.4 
 - @loadable/server: ^5.16.4 => 5.16.4 
 - @loadable/webpack-plugin: ^5.15.2 => 5.15.2 
 - webpack: ^5.90.3 => 5.90.3 
```

I'll get a reproduction repo up ASAP.

**Comment by theKashey** [comment]
😭 that should not be required

**Comment by theKashey** [comment]
Thank you for the timely report. The root cause is related only to lodash as it a little specific package without any entry points defined. Easier to fix than verify the change

**Comment by wojtekmaj** [comment]
Looks like loadable components have a "faux ESM" build that causes some issues. My guess is that other imports may be also required to be fully specified, but Weboack simply stopped in the first one. I can investigate.

**Comment by theKashey** [comment]
Fixed in @loadable/server@5.16.5
👏 @wojtekmaj for such a rapid response

**Comment by wojtekmaj** [comment]
Hopefully that will be enough to make it work :) If not, you know who to @ :D
